monkey/monkey_island/cc/ui/src/services/configuration/exploiterClasses.js

```diff
@@ -61,8 +59,8 @@ export const EXPLOITERS = {
             'enum': ['SMBExploiter'],
             'title': 'SMB Exploiter',
             'safe': true,
-            'info': 'Brute forces using credentials provided by user and' +
-            ' hashes gathered by mimikatz.',
+            'info': 'Attempts to brute-force SMB using credentials provided by the user and' +
+            ' hashes gathered by Mimikatz.',
```

Comment:
A lot of these descriptions are not perfect. Credentials might come from any source, I'm not sure we need to/are able to explain where they come from here

Reply:
We could say "using all known credentials" or something like that?

Reply:
I think that can be implied from the context. Maybe we should just delete the credentials part?